spyder/config/main.py

```diff
@@ -94,6 +94,7 @@
               'cursor/width': 2,
               'completion/size': (300, 180),
               'report_error/remember_token': False,
+              'last_visible_toolbars': [],

```

Comment:
If ToolBar is a plugin now, it deserves it's own configuration section. So why are you adding this option to `main`?

Reply:
Fixed.